processor/probabilisticsamplerprocessor/tracesprocessor_test.go

```diff
@@ -284,9 +365,16 @@ func Test_tracesamplerprocessor_SpanSamplingPriority(t *testing.T) {
 		},
 	}
 	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
+		t.Run(fmt.Sprint(tt.name), func(t *testing.T) {
+
 			sink := new(consumertest.TracesSink)
-			tsp, err := newTracesProcessor(context.Background(), processortest.NewNopCreateSettings(), tt.cfg, sink)
+			cfg := &Config{}
+			if tt.cfg != nil {
```

Comment:
do you have a new test case  where the config is nil?

Reply:
I removed this != nil test because as you noticed, none of the tests had a nil Config.